src/main/java/org/fundacionjala/katabank/BankORC.java

```diff
@@ -0,0 +1,72 @@
+package org.fundacionjala.katabank;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Class to format the number
+ */
+public class BankORC {
+
+    private static final Map<Integer, String> numberData = new HashMap<Integer, String>();
+
+    static {
+        numberData.put(1, "     |  |");
+        numberData.put(2, " _  _||_ ");
+        numberData.put(3, " _  _| _|");
+        numberData.put(4, "   |_|  |");
+        numberData.put(5, " _ |_  _|");
+        numberData.put(6, " _ |_ |_|");
+        numberData.put(7, " _   |  |");
+        numberData.put(8, " _ |_||_|");
+        numberData.put(9, " _ |_| _|");
+        numberData.put(0, " _ | ||_|");
+    }
+
+    /**
+     * Method to Parse the number read from the file
+     * @param number the string number
+     * @return the parse number
+     */
+    public static String parseNumber(String number) {
+        String numberLine1 = number.substring(0, 27);
+        String numberLine2 = number.substring(27, 54);
+        String numberLine3 = number.substring(54);
+        String result;
```

Comment:
This declaration should be inside for loop
remember. the variables should be defined in the near scope.